Cargo.toml

```diff
@@ -1,6 +1,7 @@
 [workspace]
 resolver = "2"
 members = [
+    "cargo-paradedb",
```

Comment:
why name it cargo-paradedb?

Reply:
When you name a crate `cargo-<crate>` and you `cargo install` it, you can call it globally with `cargo <crate>`.

I'm finding that quite handy working in an EC2 instance right now, because I can run these tools without needing to fiddle around with an absolute/relative path to a binary. For example, generating the elasticsearch corpus data can be done interactively with:

```
cargo install --git https://github.com/paradedb/paradedb.git cargo-paradedb
cargo paradedb generate-benchmark-corpus --out /mnt/benchmark-data
```

It's nice for this to all be easily repeatable as I run in new instances, locally, or anywhere else.

Reply:
Makes sense, thanks for explaining :)